polkadot-parachains/launch-runtime/src/lib.rs

```diff
@@ -99,10 +98,11 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("encointer-parachain"),
 	impl_name: create_runtime_str!("encointer-parachain"),
 	authoring_version: 1,
-	spec_version: 0,
+	spec_version: 1,
```

Comment:
launch should stay at 0, because anyway, it will always be a genesis config, not a runtime upgrade (I think)

Reply:
otherwise it could overtake the real runtime one day and the runtime upgrade could fail

Reply:
There was a problem that the node misinterpreted that the native runtime is valid when syncing with kusama. This lead to a storage_root mismatch error. I needed to increase the `spec_version` to fix this error.

I don't know if `execution=wasm` might solve this issue.

Why do we care if it might overtake the real runtime code? We can also just increment the spec version there?

Reply:
Yes. Ok